rules/swsssdk-py2.mk

```diff
@@ -1,7 +1,10 @@
-# swsssdk python2 wheel
+# Python 2 packages shouldn't be built for Bullseye
+ifneq ($(BLDENV), bullseye)
```

Comment:
I believe this will be applicable to Bullseye and all future versions. It is not convenient
1. to include every future version names
2. to repeat in every py2 rule file

Suggest centralize the check and share a variable like ENABLE_PY2_WHEEL

Reply:
My thinking is that once the containers are upgraded to Bullseye, the code to build the Python 2 packages would be entirely removed. I can still define a central variable if you want.

Reply:
We still have jessie in the codebase. So your think is a long roadmap. It does not harm to have a central variable